.github/workflows/CD.yml

```diff
@@ -495,72 +486,73 @@ jobs:
         run: |
           aws ecr-public batch-delete-image --repository-name public.ecr.aws/$ECR_REPO --image-ids imageTag=${{ github.event.inputs.version }} --region us-east-1
 
-      - name: Login Dockerhub
-        uses: docker/login-action@v1
-        with:
-          username: "${{ secrets.DOCKERHUB_RELEASE_USERNAME }}"
-          password: "${{ secrets.DOCKERHUB_RELEASE_TOKEN }}"
-
-      - name: Delete version image from docker
-        run: |
-          dockerhub_username=${{ secrets.DOCKERHUB_RELEASE_USERNAME }} dockerhub_password=${{ secrets.DOCKERHUB_RELEASE_TOKEN }} \
-          version=${{ github.event.inputs.version }} \
-          image_namespace=$$IMAGE_NAMESPACE image_name=$IMAGE_NAME \
-          bash tools/release/image-binary-release/delete-docker-image-tag.sh
-
 
   release-latest-image:
     runs-on: ubuntu-latest
-    needs: [s3-release-validation-1, s3-release-validation-2, s3-release-validation-3,release-validation-ecs, release-validation-eks]
+    needs: [s3-release-validation-1, s3-release-validation-2, s3-release-validation-3,release-validation-ecs, release-validation-eks, release-checking]
     steps:
       #Since the tools in workflow are always up-to-date with the github workflow, we don't need to use the tools in workflows from the commited sha
       #but using the tools in workflows from the branch triggered with workflow_dispatch.
       - uses: actions/checkout@v2
 
-      - name: Restore cached packages
+      - name: Cache if success
+        id: release-latest-image
         uses: actions/cache@v2
         with:
-          key: "${{ env.PACKAGE_CACHE_KEY }}"
-          path: build/packages
-
-      - name: Load Image
-        run: docker load < build/packages/$IMAGE_NAME.tar
+          path: |
+            VERSION
+          key: release-latest-image-${{ github.run_id }}
 
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
+      - name: Restore cached packages
```

Comment:
What do you need the cached packages for? This job is just for retagging images isn't it?

Reply:
You are right thought. The last commit takes the VERSION from build/packages/VERSION so that's why it needs a ```restore cached packages``` step. However, it is not needed here anymore so forget to delete it.